developer/openshift/gitops/argocd/pipeline-service-storage.yaml

```diff
@@ -22,8 +22,6 @@ spec:
       selfHeal: true
     syncOptions:
       - CreateNamespace=true
-      # workaround to make working both minio: newer and old 4.5.4
```

Comment:
Why did you remove this? I don't think that you will pass ci test without this workaround

Reply:
@Mo3m3n did you address @AndrienkoAleksandr 's question elsewhere ?  If not, can you clarify why this was removed?

thanks

Reply:
Actually I have mistakenly removed this  line and I have not seen this comment, I will fix it  in a subsequent PR that aims get tests to green again in main

Reply:
@gabemontero @AndrienkoAleksandr can you provide more context about this ?
I am willing to fix this and provide a more detailed explanation
>  workaround to make working both minio: newer and old 4.5.4

Can we at least tell which minio part is going to fail and possibly why?

Reply:
I'll defer to @AndrienkoAleksandr to answer your question @Mo3m3n - he was the one who investigated / put in this workaround.

I had just caught the fact that this PR merged without resolving this thread.